### PR TITLE
adding a link for the quickstart walkthrough

### PR DESCRIPTION
**Pull request by joshuafernandes**
## Pull Request Description
Added a link for the quickstart walkthrough

### CODE REVIEW

File: docs/Tutorials/Quorum-Dev-Quickstart.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ description: Helps to generate local GoQuorum and Ethereum blockchain networks.
 
 ## Usage
 
+You can follow this [walkthrough](https://consensys.net/quorum/products/guides/getting-started-with-consensys-quorum/)
```
Comment (alexandratran) [inline docs/Tutorials/Quorum-Dev-Quickstart.md:23]: As with my comment on the [Besu-docs PR](https://github.com/hyperledger/besu-docs/pull/725#discussion_r644265688), change "walkthrough" to "guide" or "walk-through", and remove the trailing space at the end of this line.

File: docs/Tutorials/Quorum-Dev-Quickstart.md [hunk 2]
```diff
@@ -29,4 +32,4 @@ npx quorum-dev-quickstart
 Follow the prompts displayed.
 
 When installation is complete, refer to `README.md` in the installation directory for more information
-on your test network.
+on your test network. Optionally refer to the walkthrough above
```
Comment (alexandratran) [inline docs/Tutorials/Quorum-Dev-Quickstart.md:35]: "Optionally, refer to the [guide/walk-through] previously mentioned."

File: docs/Tutorials/Quorum-Dev-Quickstart.md (hunk 1 above)
Comment (joshuafernandes) [reply docs/Tutorials/Quorum-Dev-Quickstart.md]: done

File: docs/Tutorials/Quorum-Dev-Quickstart.md (hunk 2 above)
Comment (joshuafernandes) [reply docs/Tutorials/Quorum-Dev-Quickstart.md]: done

File: docs/Tutorials/Quorum-Dev-Quickstart.md [hunk 3]
```diff
@@ -29,4 +32,4 @@ npx quorum-dev-quickstart
 Follow the prompts displayed.
 
 When installation is complete, refer to `README.md` in the installation directory for more information
-on your test network.
+on your test network. Optionally, refer to the walk-through above.
```
Comment (alexandratran) [inline docs/Tutorials/Quorum-Dev-Quickstart.md:35]:
```suggestion
on your test network. Optionally, refer to the previously mentioned walk-through.
```

**Review by alexandratran — APPROVED** [review]

**Pull request merged**